src/views/pages/EventPage.js

```diff
@@ -3,10 +3,24 @@ import 'styles/Event.css';
 function EventPage() {
     return (
         <div className="Event">
-            <h1>lunch</h1>
+            <p className="Event-date">FRIDAY, APRIL 22</p>
+            <h1 className="First-event">meet + greet</h1>
+            <p className="Event-title">CHOPTANK BALLROOM</p>
+            <p>7 pm - 8 pm</p>
+            <p className="Last-event-info">mingle and enjoy dinner</p>
+            <h1 className="First-event">jaggo</h1>
```

Comment:
What are the list of events that is being planned? We should consolidate so the same events that show here, show up for the invitation/rsvp page